nebula-logger/core/main/log-management/lwc/relatedLogEntries/relatedLogEntries.html

```diff
@@ -5,7 +5,7 @@
 
 <template>
     <template if:true={showComponent}>
-        <lightning-card title={title} icon-name={queryResult.tabIcon} class="log-entry-card">
+        <lightning-card title={title} icon-name="custom:custom83" class="log-entry-card">
```

Comment:
nice

Reply:
The old code for determining the tab icon was so clunky, and apparently stopped working at some point, so this seemed like one of those rare exceptions where hardcoding a string was better 😅